x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -8,24 +8,57 @@ import * as React from 'react';
 import ReactDOM from 'react-dom';
 import { CoreStart, AppMountParameters } from 'kibana/public';
 import { I18nProvider, FormattedMessage } from '@kbn/i18n/react';
+import { History, createBrowserHistory } from 'history';
```

Comment:
I don't think you need these if you use `<BrowserRouter />` component from `react-router-dom` instead of `<Router />`

Reply:
good call, changed out